pandas/tests/frame/test_reshape.py

```diff
@@ -1273,6 +1273,18 @@ def test_stack_timezone_aware_values():
     tm.assert_series_equal(result, expected)
 
 
+def test_stack_empty_frame():
+    tm.assert_series_equal(
```

Comment:
use

result=
expected=
tm.assert_series_equal (or frame)

pls parameterize these cases

add a comment with the issue number